hub/core/chunk_engine/write.py

```diff
@@ -14,11 +15,19 @@
 from hub.util.array import normalize_and_batchify_shape
 
 
+def write_tensor_meta(key: str, storage: StorageProvider, meta: dict):
+    storage[get_meta_key(key)] = pickle.dumps(meta)
+
+
+def write_dataset_meta(storage: StorageProvider, meta: dict):
+    storage[META_FILENAME] = pickle.dumps(meta)
+
+
 def write_array(
     array: np.ndarray,
     key: str,
-    chunk_size: int,
     storage: StorageProvider,
+    chunk_size: int = 16 * MB,
```

Comment:
can we put a `DEFAULT_CHUNK_SIZE` in `constants.py`?

Reply:
@McCrearyD 
There's chunk sizes thrown all around. I'll add the constant, but it's on you to implement.